This should be the exact same PR, I think

candywal opened the pull request.

candywal committed:
Merge pull request #1 from supermemo/master

editWiki Page

candywal committed:
The Images.

candywal committed:
Editwiki PAge

candywal committed:
Added link to page

alexis- commented on docs/_sidebar.md:
```diff
@@ -4,4 +4,5 @@
 - Reference manual
-- [**SuperMemo Assistant**](/sma/ ':ignore')
\ No newline at end of file
+  - [How to add to the wiki](qs-editwiki.md)  
```
`Contributing to the wiki` would be more adequate, since a user might also want to edit, or delete from it

alexis- committed:
Update qs-editwiki.md

Made several changes and added a prequel to require contributors to validate their intentions with us before putting in any work

alexis- commented on docs/qs-editwiki.md:
```diff
@@ -0,0 +1,103 @@
+
+2. Find the file you want to edit. For example, if you are looking to edit the installation page, you would select the `qs-installation.md` page.
+
+> [!TIP]
```
You can use this format 
```
> [!NOTE]
> Your text goes here
```

instead of `> HINT!`.

This will create a coloured rectangle that puts your text into context.
https://github.com/zanfab/docsify-plugin-flexible-alerts

Try it next time and see what looks best.

alexis- reviewed (changes requested):
I have edited the qs-editwiki.md file as reviewing all the changes was too complicated.
I have left a tip for you there.

Can you please:
- Delete the `blank` file
- Change the title of the page in the side bar

This is good work and will be very, very useful. Thank you

candywal committed:
Update _sidebar.md

candywal committed:
Delete blank

ZenZate commented:
Made the changes.

alexis- reviewed (approved):
OK. Accepting PR to a new branch. I will merge the changes when I am back and can view the changes in the browser on my computer.
